.github/workflows/pr-test-multicluster.yml

```diff
@@ -38,12 +38,15 @@ jobs:
           wget https://github.com/kubestellar/kubeflex/releases/download/v0.4.2/kubeflex_0.4.2_linux_amd64.tar.gz
           tar -xvf kubeflex_0.4.2_linux_amd64.tar.gz bin/kflex
           mv bin/kflex /usr/local/bin
-          rm -fr bin kubeflex_0.4.2_linux_amd64.tar.gz 
+          rm -fr bin kubeflex_0.4.2_linux_amd64.tar.gz
+          wget https://github.com/kubestellar/ocm-transport-plugin/archive/refs/tags/v0.1.0-rc1.tar.gz
```

Comment:
Please move that to setup-kubestellar.  As you build there that is your  actual install.  You can then also delete the source directory after the build

Reply:
I see that https://github.com/kubestellar/ocm-transport-plugin/releases/tag/v0.1.0-rc1 also has archives of built stuff. Do we really need to have contributors and CI repeat the build?

Reply:
@MikeSpreitzer That was my comment above (https://github.com/kubestellar/kubestellar/pull/1780#discussion_r1494625925) 
@nirrozenbaum point was that he wanted to be able to test new changes in the base transport so he needs to rebuild it. 
I think this is a problematic dependency and it's not a reasonable requirement.    I'm OK with postponing  this to follow-up PR

Reply:
right. an executable is pointing to a specific version of KubeStellar.
assuming current PR changes transport behavior we want to test the NEW behavior.
for this we must get the source of ocm plugin, make sure it points to the NEW code that is currently being tested and build the executable/image.

a pre-build executable or image will not test the new changes of KubeStellar transport.
this is indeed problematic but let's solve this in a follow up PR (depends on the decision of how we solve the cycle) to unblock other PRs and issues

Reply:
/unhold

Reply:
(A) @nirrozenbaum : I think the purpose of the two existing E2E tests written in bash is twofold: (1) test that the local copy of the ks/ks repo works, and (2) show contributors how to exercise the `main` branch of ks/ks. My understanding of our plans is that we intend to make a release (@pdettori is working on that today) of ks/ks that does NOT use pluggable transport. If this is correct, then that should be completed before a change like this PR should be merged.

(B) I think we do need to settle the issue of how to handle the dependency cycle. As I mentioned in my discussion in Slack, I think that hacking around with both `main`s in absence of a principled approach can get us stuck.

Reply:
@MikeSpreitzer I think your (A) is not relevant here because this code is not executed without bringing up the controller . I don't think it will be problematic for @pdettori to have this (dead) code in the release. 
In any case I don't mind waiting to after 0.20.0

Reply:
agree with Ezra. since this is an executable and whoever runs 0.20 will use helm to install it, the helm will not install transport controller.

Reply:
@ezrasilvera : I am not sure I understand what you mean by "this code is not executed without bringing up the controller". A contributor reading these E2E tests and doing what is in them (see my A.2) will build and bring up the transport controller, right?

@nirrozenbaum, I am surprised here. I thought you favored the staggered release approach --- which this is not. I expect @pdettori to have ks 0.20.0-rc1 very soon.

Reply:
Correction: ks 0.20.0-rc1 is there now.

Reply:
np from my side to wait for Paolo's release.